src/components/RenderList.vue

```diff
@@ -9,17 +9,17 @@
           <strong>{{ingredient.name}}</strong>
           <span>Weight:</span>  
           <input placeholder="Edit" v-model.number="ingredient.quantity" type ="number" />
-          <span>Price:</span>
+          <span>Price/Kg:</span>
           <input placeholder="Edit" v-model.number="ingredient.price" type ="number" />
           <span>VAT:</span>
           <select v-model.number="ingredient.vat" >
             <option  disabled value="">Select VAT</option>
-            <option>5</option>
-            <option>8</option>
-            <option>23</option>
+            <option value="1.05">5</option>
+            <option value="1.08">8</option>
+            <option value="1.23">23</option>
```

Comment:
Please make it dynamic, move it to `data > vatOptions`

Reply:
✔️